docs/source/index.rst

```diff
@@ -8,12 +8,12 @@ It provides I/O, signal and data processing functions, datasets,
 model implementations and application components.
 
 .. note::
-    Starting with version 2.8, we are refactoring TorchAudio to transition it
+    Starting with version 2.8, TorchAudio has transitioned
     into a maintenance phase. As a result:
 
-    - Some APIs are deprecated in 2.8 and will be removed in 2.9.
+    - Some APIs were deprecated in 2.8 and removed in 2.9.
     - The decoding and encoding capabilities of PyTorch for both audio and video
-      are being consolidated into TorchCodec.
+      have been consolidated into TorchCodec.
```

Comment:
Below we are including the `build.*` stuff with lots of references to ffmpeg. We should remove those in a follow-up PR.

Reply:
Above there's a mention that torchaudio supports `IO` stuff which isn't reallt the case anymore. We should edit that, and edit the same statement in the main `README.md` in a follow-up PR.

Reply:
I removed the reference to IO. By "removing these in a follow up PR", do you mean removing the special ffmpeg-specific install instructions for each platform?

Reply:
yes